ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -261,6 +272,13 @@ def _connection_cache(self) -> Dict[_ConnectionKey, Connection]:
                 cache[key] = value
         return cache
 
+    @property
+    def multiplexer_session_info(self) -> Sequence[MultiplexerSessionInformation]:
+        """Multiplexer session information object."""
+        if not self._multiplexer_session_info:
+            raise ValueError("No multiplexer session(s) exist.")
```

Comment:
This error seems overly harsh. It makes it harder for the service to ask whether there is a multiplexer or not.

The corresponding `session_info` property doesn't have this error. The multi-session version returns an empty list when there are no sessions.

Reply:
Sure. Removed the check and updated the test to verify if an empty list is returned.